ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -0,0 +1,63 @@
+<h3 class="title is-4">Quick actions</h3>
+
+<div class="has-bottom-margin-m">
+  <h4 class="title is-6">Secrets engines</h4>
+  <SearchSelect
+    @id="secrets-engines-select"
+    @options={{this.mountOptions}}
+    @selectLimit="1"
+    @disallowNewItems={{true}}
+    @fallbackComponent="input-search"
+    @onChange={{this.handleSearchEngineSelect}}
+    @placeholder="Type to select a mount..."
```

Comment:
same comment here about ellipsis. If it's on the designs, I feel like it's worth clarifying.

Reply:
That makes sense to me! I'll remove the ellipsis for now, and ask design about it!